### PR TITLE
 Add function to escape string for translation in the string translation panel

### PR DESCRIPTION
**Pull request by jegbagus**
this commit related to ticket #82

### CODE REVIEW

File: include/api.php [hunk 1]
```diff
@@ -130,6 +130,18 @@ function pll__( $string ) {
...
 /**
+ * Translates a string ( previously registered with pll_register_string ) and escapes it for safe use in HTML output.
+ * 
+ * @since 2.1
+ * 
+ * @param string $string the string to translate
+ * @return string translation in the current language
+ */
+function plll_esc_htlm__ ( $string ) {
+	return esc_html__( pll__( $string ) );
```
Comment (grappler) [inline include/api.php]: You are missing the text domain here. You want to do `return esc_html( pll__( $string ) );`.

File: include/api.php [hunk 2]
```diff
@@ -130,6 +130,18 @@ function pll__( $string ) {
...
 
 /**
+ * Translates a string ( previously registered with pll_register_string ) and escapes it for safe use in HTML output.
+ * 
+ * @since 2.1
+ * 
+ * @param string $string the string to translate
+ * @return string translation in the current language
+ */
+function plll_esc_htlm__ ( $string ) {
```
Comment (grappler) [inline include/api.php]: There is a "l" too many in `plll` and html is spelled incorectly

Comment (Chouby) [inline include/api.php]: Could you please rename the function to pll_esc_html__ including in the function where it is used below?

**Comment by grappler** [comment]
`pll_esc_attr_e()` & `pll_esc_attr__()` should be included too if `pll_esc_html__()` is going to added.

**Comment by Chouby** [comment]
Hi!

I ma ok to introduce these new functions. We could also add the corresponding functions for `esc_attr() `

**Comment by jegbagus** [comment]
hi @grappler thank you for correction. Also add pll_esc_attr (without _e and __). WordPress it self not having esc_attr__ or esc_attr_e function. so i think we don't need to have it also. 

Thank you

File: include/api.php [hunk 3]
```diff
@@ -130,6 +130,30 @@ function pll__( $string ) {
...
 
 /**
+ * Translates a string ( previously registered with pll_register_string ) and escapes it for safe use in HTML output.
+ * 
+ * @since 2.1
+ * 
+ * @param string $string the string to translate
+ * @return string translation in the current language
+ */
+function pll_esc_htlm__ ( $string ) {
```
Comment (grappler) [inline include/api.php]: There is a space between the function name and the opening bracket. HTML has still a typo.

File: include/api.php [hunk 4]
```diff
@@ -141,6 +165,18 @@ function pll_e( $string ) {
...
+ *
+ * @since 2.1
+ *
+ * @param string $string the string to translate
+ */
+function pll_esc_html_e( $string ) {
+	echo pll_esc_htlm__( $string );
+}
+
+
```
Comment (grappler) [inline include/api.php]: There are two line breaks instead of one.

**Comment by grappler** [comment]
> WordPress it self not having esc_attr or esc_attr_e function

I am not sure what you mean. You can see these two function below.
https://developer.wordpress.org/reference/functions/esc_attr_e/
https://developer.wordpress.org/reference/functions/esc_attr__/

**Comment by Chouby** [comment]
I renamed `pll_esc_attr()` to `pll_esc_attr__()` to follow the same pattern as similar functions. I also added `pll_esc_attr_e()`

**Pull request merged**